src/DotNetWorker.ApplicationInsights/Initializers/AppServiceOptionsInitializer.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using Microsoft.Azure.Functions.Worker.ApplicationInsights.Initializers;
+using Microsoft.Extensions.Options;
+
+internal class AppServiceOptionsInitializer : IConfigureOptions<AppServiceOptions>
+{
+    internal const string AzureWebsiteName = "WEBSITE_SITE_NAME";
+    internal const string AzureWebsiteSlotName = "WEBSITE_SLOT_NAME";
+    internal const string AzureWebsiteCloudRoleName = "WEBSITE_CLOUD_ROLENAME";
+    internal const string DefaultProductionSlotName = "production";
+
+    internal static string[] EnvironmentVariablesToMonitor = [AzureWebsiteName, AzureWebsiteSlotName, AzureWebsiteCloudRoleName];
```

Comment:
Does this style of collection initializer works in net6/net7? Should we use `new [` ?

Reply:
build failed -- so yeah :-)